### ISSUE TITLE
manual/scalaGuide/_Sidebar.md

### ISSUE THREAD

**Issue by garbagetown** [playframework-ja/translation-project#980]
- https://github.com/playframework-ja/translation-project/tree/2.3.5/manual/scalaGuide/_Sidebar.md
- diffs between [2.2.0/manual/scalaGuide/_Sidebar.md](https://github.com/playframework-ja/translation-project/tree/2.2.0/manual/scalaGuide/_Sidebar.md) and [2.3.5/manual/scalaGuide/_Sidebar.md](https://github.com/playframework-ja/translation-project/tree/2.3.5/manual/scalaGuide/_Sidebar.md).

```
--- /Users/garbagetown/Desktop/2.2.0/manual/scalaGuide/_Sidebar.md  2013-09-19 10:11:22.000000000 +0900
+++ //Users/garbagetown/Desktop/2.3.5/manual/scalaGuide/_Sidebar.md 2014-07-28 12:26:38.000000000 +0900
@@ -1,3 +1,4 @@
+<!--- Copyright (C) 2009-2013 Typesafe Inc. <http://www.typesafe.com> -->
 ### Main concepts

 - [[HTTP programming | ScalaActions]]
@@ -13,7 +14,8 @@
 - [[Integrating with Akka | ScalaAkka]]
 - [[Internationalization | ScalaI18N]]
 - [[The application Global object | ScalaGlobal]]
-- [[Testing your application | ScalaTest]]
+- [[Testing your application | ScalaTestingYourApplication]]
+- [[Logging|ScalaLogging]]

 ### Advanced topics

@@ -22,7 +24,3 @@
 - [[HTTP Filters | ScalaHttpFilters]]
 - [[Dependency Injection | ScalaDependencyInjection]]
 - [[Extending Play|ScalaPlugins]]
-
-### Tutorials
-
-- [[Your first application | ScalaTodoList]]

```

**Comment by garbagetown** [comment]
:hand:

**Comment by garbagetown** [comment]
merged at #1049
